src/constants.ts

```diff
@@ -6,3 +8,6 @@ export const ROUTES: Record<IRouteKey, string> = {
   room: '/room/:id',
   game: '/game/:id'
 };
+
+export const API_ENDPOINT = 'localhost:8080';
```

Comment:
nit: If deploying the app to a remote server, this should come from a `.env` file as the server path may change based on env